es74x/src/main/scala/tech/beshu/ror/es/rradmin/RRAdminResponse.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.es.rradmin
+
+import org.apache.logging.log4j.scala.Logging
+import org.elasticsearch.action.ActionResponse
+import org.elasticsearch.common.io.stream.StreamOutput
+import org.elasticsearch.common.xcontent.{ToXContent, ToXContentObject, XContentBuilder}
+import tech.beshu.ror.adminapi.AdminRestApi
+
+class RRAdminResponse(response: Either[Throwable, AdminRestApi.AdminResponse])
+  extends ActionResponse with ToXContentObject with Logging {
+
+  def this(response: AdminRestApi.AdminResponse) {
+    this(Right(response))
+  }
+  override def writeTo(out: StreamOutput): Unit = {???}
```

Comment:
Seems that we need to provide some implementation in new version of ES for this method. IMO we should do this for previous version. So, maybe when we do that for es73x module, there won't be necessary to create new es74x? WDYT? Can it be done or there are any other incompatibilities?

Reply:
There are. Some constructors are getting `StreamInput`